### PR TITLE
task/frontend58 + toast notification

### PR DESCRIPTION
**Pull request by so5tmaker**

### CODE REVIEW

**Comment by so5tmaker** [comment]
[Implement toaster notifications](https://trello.com/c/xh35nsVE/58-implement-toaster-notifications)

File: frontend/src/components/common/notifications/notification.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import cn from 'clsx';
+
+import styles from './styles.module.scss';
+
+interface iNotificationParams {
+  type: NOTIFICATION_TYPE;
+  iconName: string;
+  title: string;
+  message: string;
+}
```
Comment (kyrylo-burmelov) [inline frontend/src/components/common/notifications/notification.tsx]: Follow standards for interface name. It should start from capital letter and the name should be `Props`

File: frontend/src/components/common/notifications/notification.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+const Notification = ({ type, iconName, title, message }: iNotificationParams): JSX.Element => {
```
Comment (kyrylo-burmelov) [inline frontend/src/components/common/notifications/notification.tsx]: Follow standard, use `FC` it for it

File: frontend/src/components/common/notifications/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  display: inline-block;
+  width: 100%;
+  padding: 8px 15px;
+}
+
+.title,
+.message {
+  margin-top: 5px;
+  margin-bottom: 5px;
+  color: #ffffff;
```
Comment (kyrylo-burmelov) [inline frontend/src/components/common/notifications/styles.module.scss]: use vars for all colors

File: package.json [hunk 4]
```diff
@@ -57,5 +57,9 @@
     "*.{ts,tsx,json,scss}": "prettier --write",
     "{shared,backend/core,backend/push,frontend}/**/*.{ts,tsx}": "npm run lint:js",
     "frontend/**/*.scss": "npm run lint:css"
+  },
+  "dependencies": {
+    "@types/react-notifications-component": "^3.1.1",
+    "react-notifications-component": "^4.0.1"
```
Comment (kyrylo-burmelov) [inline package.json]: react-notifications-component must be installed to frontend only

types must be installed as dev dependency

**Review by kyrylo-burmelov — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (so5tmaker) [reply package.json]: Done!

File: frontend/src/components/common/notifications/styles.module.scss (hunk 3 above)
Comment (so5tmaker) [reply frontend/src/components/common/notifications/styles.module.scss]: Done!

File: frontend/src/components/common/notifications/notification.tsx (hunk 2 above)
Comment (so5tmaker) [reply frontend/src/components/common/notifications/notification.tsx]: Done!

File: frontend/src/components/common/notifications/notification.tsx (hunk 1 above)
Comment (so5tmaker) [reply frontend/src/components/common/notifications/notification.tsx]: Done!

**Comment by so5tmaker** [comment]
[Implement toaster notifications](https://trello.com/c/xh35nsVE/58-implement-toaster-notifications)

File: frontend/package.json [hunk 5]
```diff
@@ -17,11 +17,13 @@
     "@hookform/error-message": "2.0.0",
     "@hookform/resolvers": "2.9.6",
     "@reduxjs/toolkit": "1.8.3",
+    "@types/react-notifications-component": "^3.1.1",
```
Comment (kyrylo-burmelov) [inline frontend/package.json]: Again! Types must be dev dependencies

**Review by kyrylo-burmelov — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 5 above)
Comment (so5tmaker) [reply frontend/package.json]: Done!

File: frontend/package.json [hunk 6]
```diff
@@ -32,6 +33,7 @@
     "@types/react": "18.0.15",
     "@types/react-dom": "18.0.6",
     "@types/redux-persist": "4.3.1",
+    "@types/react-notifications-component": "^3.1.1",
```
Comment (kyrylo-burmelov) [inline frontend/package.json]: discussed in the slack

**Review by kyrylo-burmelov — CHANGES_REQUESTED** [review]

**Review by kyrylo-burmelov — APPROVED** [review]

**Pull request merged**